src/Components/Votes.jsx

```diff
@@ -0,0 +1,57 @@
+import IconButton from "@mui/material/IconButton";
+import FavoriteSharpIcon from "@mui/icons-material/FavoriteSharp";
+import Stack from "@mui/material/Stack";
+import { patchArticle } from "../api";
+import { useState } from "react";
+
+const Votes = ({ article, setArticle }) => {
+  const [beenClicked, setbeenClicked] = useState(false);
+  const [buttonColour, setButtonColour] = useState("notLiked");
+
+  const setToLiked = () => {
+    setArticle((currArticle) => {
+      return { ...currArticle, votes: currArticle.votes + 1 };
+    });
+    setbeenClicked(true);
+    setButtonColour("liked");
+  };
+
+  const removeLike = () => {
+    setArticle((currArticle) => {
+      return { ...currArticle, votes: currArticle.votes - 1 };
+    });
+    setbeenClicked(false);
+    setButtonColour("notLiked");
+  };
+
+  const changeVotes = () => {
+    if (beenClicked) {
+      patchArticle(article.article_id, { inc_votes: -1 }).catch((err) => {
+        setToLiked();
+      });
+      removeLike();
+    } else {
+      patchArticle(article.article_id, { inc_votes: +1 }).catch((err) => {
+        removeLike();
+      });
+      setToLiked();
+    }
+  };
```

Comment:
Love love love the Symbol and color change ! However, it would be nice to inform the user about a failed up vote if it didnt go through (e.g. network connection cut off)